backend/utils/eeljokes.py

```diff
@@ -0,0 +1,14 @@
+from flask_sqlalchemy import SQLAlchemy
+
+
+db = SQLAlchemy()
+
+class EelJokes(db.Model):
+    __tablename__ = 'EelJokes'
+    JokeID = db.Column(db.Integer, primary_key=True)
+    JokeText = db.Column(db.Text, nullable=False)
+    JokeTextLine2 = db.Column(db.Text, nullable=True)
+
+    def get_joke_data(self):
+        joke_data = {"JokeID": self.JokeID, "JokeText": self.JokeText, "JokeTextLine2": self.JokeTextLine2}
+        return joke_data
```

Comment:
```suggestion
        return {"JokeID": self.JokeID, "JokeText": self.JokeText, "JokeTextLine2": self.JokeTextLine2}
```

nit: If you aren't doing anything to object but simply returning it as is, u don't need to assign it to a temp var like this.